buildSrc/src/main/groovy/org/apache/geode/gradle/plugins/DependencyConstraints.groovy

```diff
@@ -67,6 +67,7 @@ class DependencyConstraints implements Plugin<Project> {
     deps.put("junit.version", "4.13.2")
     deps.put("junit-jupiter.version", "5.7.2")
     deps.put("cglib.version", "3.3.0")
+    deps.put("gradle-tooling-api.version", "5.1.1")
```

Comment:
I don't like that this is now separated from the comment that explains why it's such an old version